spec/requests/api/v1/courses_spec.rb

```diff
@@ -0,0 +1,5 @@
+require 'swagger_helper'
+# rubocop:disable Lint/EmptyBlock
+RSpec.describe 'api/v1/courses', type: :request do
+end
```

Comment:
Please add tests for this section. Tests ought to be comprehensive

Reply:
Thank you. I will try to add more comprehensive tests.